hiredis.h

```diff
@@ -156,6 +156,7 @@ struct redisSsl;
 #define REDIS_OPT_REUSEADDR 0x02
 #define REDIS_OPT_PREFER_IPV4 0x04
 #define REDIS_OPT_PREFER_IPV6 0x08
+#define REDIS_OPT_PREFER_UNSPEC (REDIS_OPT_PREFER_IPV4 | REDIS_OPT_PREFER_IPV6)
```

Comment:
Maybe "IP" should be included in the macro name?

```suggestion
#define REDIS_OPT_PREFER_IP_UNSPEC (REDIS_OPT_PREFER_IPV4 | REDIS_OPT_PREFER_IPV6)
```